src/gui/keymap.cpp

```diff
@@ -93,6 +93,10 @@ static const KeyPatch patchMX[] = {
     {nullptr,  "<", "bslash_iso"}, {nullptr,  "-", "slash"},
 };
 
+static const KeyPatch patchPT[] = {
+    // DON'T KNOW WHAT TO DO WITH THIS
```

Comment:
This is where you replace the keys that doesn't match with the ones that are in your keyboard.

Reply:
thank you, I will try